codalab/apps/newsletter/scripts/send_emails_to_mailchimp.py

```diff
@@ -0,0 +1,27 @@
+import json
+import requests
+
+from django.contrib.auth import get_user_model
+from codalab import settings
+
+User = get_user_model()
+
+
+def run():
```

Comment:
need to make sure mailchimp settings exist. Also, catch request exceptions here too